docs/source/apps.rst

```diff
@@ -184,3 +184,11 @@ Applications
     :members:
 .. automodule:: monai.apps.detection.metrics.matching
     :members:
+
+
+
+"Reconstruction"
```

Comment:
Please follow the exact format in rst files, e.g.,

`Pathology`
-----------

.. automodule:: monai.apps.pathology.data